main/src/cgeo/geocaching/maps/google/v2/GoogleMapView.java

```diff
@@ -146,7 +148,8 @@ public void onMapReady(final GoogleMap googleMap) {
             final float bearing = cameraPosition.bearing;
             if (canDisableAutoRotate && bearing == 0.0f && Settings.getMapRotation() == Settings.MAPROTATION_AUTO) {
                 canDisableAutoRotate = false;
-                SimpleDialog.of((Activity) getContext()).setTitle(R.string.map_gm_autorotation).setMessage(R.string.map_gm_autorotation_disable).confirm((dialog, which) -> {
+                final Context context = getContext();
+                Dialogs.advancedOneTimeMessage(context, OneTimeDialogs.DialogType.MAP_AUTOROTATION_DISABLE, context.getString(R.string.map_gm_autorotation), context.getString(R.string.map_gm_autorotation_disable), "", true, null, () -> {
```

Comment:
Is there a reason why you haven't used the `Dialogs.basicOneTimeMessage()` method which just requires Context and DialogType (-> way simpler)?

Reply:
Ok, just saw that basicOneTimeMessages aren't cancelable and have the "info icon". The icon would be fine IMHO and for the cancelable flag we should edit our basic OneTimeMessage instead. I don't remember a reason why these dialogs should not be able to cancel...

Reply:
IIRC basicOneTimeMessage only displays a notice, but does not allow for starting some follow-up action on confirmation - which is something I need here.

Reply:
True, have not seen this. Then it maybe would make sense to reference title and text via `MAP_AUTOROTATION_DISABLE.messageTitle` / ``MAP_AUTOROTATION_DISABLE.messageText`. 

Or we need to create some kind of "halfAdvanced" OTM. 

But I would also be fine with merging the current state.

Reply:
I'll update the PR to using `MAP_AUTOROTATION_DISABLE.messageTitle` / `MAP_AUTOROTATION_DISABLE.messageText`